src/Estimators/CMakeLists.txt

```diff
@@ -3,19 +3,21 @@
 # GNU Lesser General Public License v2.1 or any later version.
 
 if(FRAMEWORK_COMPILE_Estimators)
-
   set(H_PREFIX include/BipedalLocomotion/Estimators)
   add_bipedal_locomotion_library(
     NAME                   Estimators
     SOURCES                src/RecursiveLeastSquare.cpp
     PUBLIC_HEADERS         ${H_PREFIX}/RecursiveLeastSquare.h
     SUBDIRECTORIES         tests
     PUBLIC_LINK_LIBRARIES  BipedalLocomotion::ParametersHandler BipedalLocomotion::System Eigen3::Eigen)
+endif()
 
+if(FRAMEWORK_COMPILE_FloatingBaseEstimators)
+  set(H_PREFIX include/BipedalLocomotion/FloatingBaseEstimators)
   add_bipedal_locomotion_library(
     NAME                   FloatingBaseEstimators
-    SOURCES                src/FloatingBaseEstimator.cpp
-    PUBLIC_HEADERS         ${H_PREFIX}/FloatingBaseEstimatorParams.h ${H_PREFIX}/FloatingBaseEstimatorIO.h ${H_PREFIX}/FloatingBaseEstimator.h
-    PUBLIC_LINK_LIBRARIES  iDynTree::idyntree-high-level iDynTree::idyntree-core iDynTree::idyntree-model BipedalLocomotion::System Eigen3::Eigen)
-
+    SOURCES                src/FloatingBaseEstimator.cpp src/InvariantEKFBaseEstimator.cpp
+    PUBLIC_HEADERS         ${H_PREFIX}/FloatingBaseEstimatorParams.h ${H_PREFIX}/FloatingBaseEstimatorIO.h ${H_PREFIX}/FloatingBaseEstimator.h ${H_PREFIX}/InvariantEKFBaseEstimator.h
+    PUBLIC_LINK_LIBRARIES  iDynTree::idyntree-high-level iDynTree::idyntree-core iDynTree::idyntree-model BipedalLocomotion::System Eigen3::Eigen
```

Comment:
``BipedalLocomotion::ParametersHandler`` seems missing

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/85/commits/434835236824d749ac2d60e93ceb6eefbc3d96e9